pipeline.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     - get: pipeline-tasks
     - get: terraform-templates
       trigger: true
-  - task: apply-terraform
+  - task: terraform-apply
```

Comment:
Did this change just to match the job name above and/or the script name below?  Trying to recall what this is in reference to.